### PR TITLE
feature (refs AB#18560): new page per original statement

### PR DESCRIPTION
**Pull request by breierDemos**
### Ticket
[ADO-18560](https://www.dev.diplanung.de/DefaultCollection/BOP-HH/_workitems/edit/18560)

This PR ensures that a new page begins for each original statement in the PDF export.

### How to review/test
Export original statements in the assessment table via PDF export with all statements, with a few statements, or with one statement.

### PR Checklist

Delete the checkbox if it doesn't apply/isn't necessary.

- [ ] Tests updated/created
- [ ] Update documentation
- [x] Link all relevant tickets
- [ ] Move the tickets on the board accordingly
- [ ] Data-Cy attributes added/updated ([conventions](https://dplan-documentation.demos-europe.eu/development/guidelines-conventions/coding-styleguides/twig_html.html#guideline-for-naming-cypress-hooks))

### CODE REVIEW

**Review by MoritzMandler — APPROVED** [review]
lgtm - tested together :)

File: client/js/components/statement/assessmentTable/DpExportModal.vue [hunk 1]
```diff
@@ -62,6 +62,25 @@
                 v-model="exportChoice.pdf.anonymous" />
             </fieldset>
 
+            <fieldset
+              v-if="options.pdf.newPagePerStn && view === 'original_statements'"
+              class="u-mb-0_5 u-pb-0_5">
+<!--              <legend-->
+<!--                class="sr-only"-->
+<!--                v-text="" />-->
```
Comment (gruenbergerdemos) [inline client/js/components/statement/assessmentTable/DpExportModal.vue:70]: Do we need this commented code here?

File: client/js/components/statement/assessmentTable/DpExportModal.vue [hunk 2]
```diff
@@ -62,6 +62,25 @@
...
+                data-cy="exportModal:newPagePerStn"
+                id="pdfNewPagePerStn"
+                :label="{
+                  bold: true,
+                  text: Translator.trans('export.newPagePerStatement')
+                }"
+                name="newPagePerStn"
+                value="newPagePerStn"
+                v-model="exportChoice.pdf.newPagePerStn"
+              />
```
Comment (gruenbergerdemos) [inline client/js/components/statement/assessmentTable/DpExportModal.vue:81]:
```suggestion
              <dp-checkbox
                id="pdfNewPagePerStn"
                v-model="exportChoice.pdf.newPagePerStn"
                data-cy="exportModal:newPagePerStn"
                :label="{
                  bold: true,
                  text: Translator.trans('export.newPagePerStatement')
                }"
                name="newPagePerStn"
                value="newPagePerStn" />
```

**Review by gruenbergerdemos — CHANGES_REQUESTED** [review]

File: client/js/components/statement/assessmentTable/DpExportModal.vue [hunk 3]
```diff
@@ -62,6 +62,25 @@
...
+<!--                v-text="" />-->
+              <dp-checkbox
+                data-cy="exportModal:newPagePerStn"
+                id="pdfNewPagePerStn"
+                :label="{
+                  bold: true,
+                  text: Translator.trans('export.newPagePerStatement')
+                }"
+                name="newPagePerStn"
+                value="newPagePerStn"
```
Comment (gruenbergerdemos) [inline client/js/components/statement/assessmentTable/DpExportModal.vue:79]: For what do we need this value?

File: client/js/components/statement/assessmentTable/DpExportModal.vue (hunk 1 above)
Comment (hwiem) [reply client/js/components/statement/assessmentTable/DpExportModal.vue]: Yes, the text was still missing. I added it here 67faa66eb3b555e13cbaefab60ab312adafbcdc8

File: client/js/components/statement/assessmentTable/DpExportModal.vue (hunk 3 above)
Comment (hwiem) [reply client/js/components/statement/assessmentTable/DpExportModal.vue]: We don't need it, removed in acf774c3a195fe8d1d192aa8ac15a25a3e0358eb

**Pull request merged**